common4.py

```diff
@@ -25,14 +19,15 @@ def run_standard_text_flow(get_intervals, length_needed, total_num_pitches, make
         file.write("\nPossible pitch outcomes using {} for the {} entered intervals: {}\n\n".format(decision, seq_length, interval_set))
         file.write(str(string_list))
         file.write("\n\n")
-        if decision.lower() == 'y' or decision.lower() == 'yes':
+        if show_binary.lower().strip() in {'y', 'yes'}:
+            file.write("\n\nSelected binary ordering was: {} which means {}\n".format(binary_order, decision))
```

Comment:
i added this so i didnt "waste" the decision variable. not sure if you want it or not, but i didnt want that per-choice explanation to go unused